crates/modelardb_storage/src/delta_lake.rs

```diff
@@ -478,20 +482,16 @@ impl DeltaLake {
     /// successfully, otherwise returns [`ModelarDbStorageError`].
     async fn write_record_batches_to_table(
         &self,
-        delta_table: DeltaTable,
+        mut delta_table_writer: DeltaTableWriter,
         record_batches: Vec<RecordBatch>,
-        partition_columns: Vec<String>,
```

Comment:
Update the doc comment for this method to no longer mention `writer_properties`, `delta_table`, and `partition_columns` and instead mention the new argument.